src/components/TopBar.js

```diff
@@ -230,143 +308,93 @@ export default _.flow(
           h(Clickable, {
             as: 'a',
             ...Utils.newTabLinkProps,
-            style: styles.nav.item,
+            style: { ...styles.nav.item, borderBottom: `1px solid ${colors.dark(0.55)}` },
             hover: { backgroundColor: colors.dark(0.55) },
             href: getConfig().jobManagerUrlRoot,
             onClick: () => this.hideNav()
           }, [
             div({ style: styles.nav.icon }, [
               icon('layers', { className: 'is-solid', size: 24 })
             ]),
-            'Your Jobs',
-            icon('pop-out', {
-              size: 12,
-              style: { marginLeft: '0.5rem' }
-            })
-          ]),
-          div({ style: { borderBottom: styles.nav.item.borderBottom, padding: '14px 0' } }, [
-            div({ style: { ...styles.nav.subItem, paddingLeft: 28 } }, [
-              div({ style: styles.nav.icon }, [
-                icon('library', { className: 'is-solid', size: 24 })
-              ]),
-              'Library'
-            ]),
-            librarySubItem('library-datasets', 'data-cluster', 'Datasets'),
-            librarySubItem('library-showcase', 'grid-chart', 'Showcase & Tutorials'),
-            librarySubItem('library-code', 'tools', 'Code & Tools')
+            'Job Manager'
           ]),
+          div({ style: { margin: '5rem' } }),
+          h(BuildDropDownSection, {
```

Comment:
This should probably be open if you are on one of the pages included in this section. Also applies to the profile section... but I guess not the Terra Support section because everything there either opens a different site in a new window or opens a modal.